docs/team/muraddurrani.md

```diff
@@ -11,12 +11,29 @@ meetings.
 
 ## Summary of Contributions
 
-* **Code Contributed**: `to be added soon`
-* **Enhancements Implemented**: `to be added soon`
-* **Contributions to the UG**: `to be added soon`
-* **Contributions to the DG**: `to be added soon`
-* **Contributions to the team-based tasks**: `to be added soon`
-* **Review/mentoring contributions**: `to be added soon`
-* **Contributions beyond the project team**: `to be added soon`
-* **Contributions to the Developer Guide (Extracts)**: `to be added soon`
-* **Contributions to the User Guide (Extracts)**: `to be added soon` 
+* **Code Contributed**: [tP Code Dashboard Link](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-02-18&tabOpen=true&tabType=authorship&zFR=false&tabAuthor=muraddurrani&tabRepo=AY2122S2-CS2103T-W12-4%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
```

Comment:
```suggestion
* **Code Contributed**: [tP Code Dashboard Link](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=muraddurrani&breakdown=true&sort=groupTitle&sortWithin=title&since=2022-02-18&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
```